index.src.html

```diff
@@ -501,7 +501,8 @@ <h3 id="dom-api">JavaScript API</h3>
       Clears data based on the values in the |options| argument.
       Returns a Promise that resolves when clearing is complete. If no
       {{StorageClearOptions/types}} are specified, all data types will be
-      cleared.
+      cleared. See [[#clear-api]] for the formal description of the clearing
+      algorithm.
```

Comment:
@mikewset Done. Yeah, I'm not yet fluent in w3c-ese... :)

@annevk I'm not sure what you mean. The section name is the only thing I can link to, and there is also nothing else in the section, so it's clear that I'm referring to the algorithm. Would you prefer not linking but instead spelling out "'Clear data for options' algorithm"?